env/AZUREPW.env

```diff
@@ -0,0 +1,298 @@
+#! /usr/bin/env bash
```

Comment:
Please remove Azure and Google additions from this PR as this PR is titled "Implemente global-workflow on AWS"